src/signals_notebook/entities/plates/cell.py

```diff
@@ -0,0 +1,43 @@
+from enum import Enum
+from typing import Generic, List, Optional, TypeVar
+from uuid import UUID
+
+from pydantic import Field
+from pydantic.generics import GenericModel
+
+
+CellContentType = TypeVar('CellContentType')
+
+
+class ColumnDataType(str, Enum):
+    NUMBER = 'NUMBER'
+    INTEGER = 'INTEGER'
+    DATE_TIME = 'DATETIME'
+    TEXT = 'TEXT'
+    LIST = 'LIST'
+    MULTI_SELECT = 'MULTI_SELECT'
+    ATTRIBUTE_LIST = 'ATTRIBUTE_LIST'
+    AUTOTEXT_LIST = 'AUTOTEXT_LIST'
+    BOOLEAN = 'BOOLEAN'
+    UNIT = 'UNIT'
+    LINK = 'LINK'
+    EXTERNAL_LINK = 'EXTERNAL_LINK'
+
+
+class CellContent(GenericModel, Generic[CellContentType]):
+    user: Optional[str]
+    value: CellContentType
```

Comment:
What do u think about this attribute?
Is it correct to set it Optional[cellvalue] = None?